examples/jdbc/src/main/java/com/clickhouse/examples/jdbc/Basic.java

```diff
@@ -1,190 +1,53 @@
-                    + "`actor_type` LowCardinality(String),"
-                    + "`actor_id` String,"
-                    + "`actor_tenant_id` LowCardinality(String),"
-                    + "`actor_tenant_name` String,"
-                    + "`actor_firstname` String,"
-                    + "`actor_lastname` String,"
-                    + "`resource_type` LowCardinality(String),"
-                    + "`resource_id` String,"
-                    + "`resource_container` LowCardinality(String),"
-                    + "`resource_path` String,"
-                    + "`origin_ip` String,"
-                    + "`origin_app_name` LowCardinality(String),"
-                    + "`origin_app_instance` String,"
-                    + "`description` String,"
-                    + "`attributes` Map(String, String)"
-                    + ")"
-                    + "ENGINE = MergeTree "
-                    + "ORDER BY (resource_container, event_type, event_subtype) "
-                    + "SETTINGS index_granularity = 8192");
-            try (PreparedStatement stmt = conn.prepareStatement(
-                    "INSERT INTO t_map SETTINGS async_insert=1,wait_for_async_insert=1 VALUES " +
-                            "(8481365034795008,1673349039830,'operation-9','a','service', 'bc3e47b8-2b34-4c1a-9004-123656fa0000','b', 'c', 'service-56','d', 'object','e', 'my-value-62', 'mypath', 'some.hostname.address.com', 'app-9', 'instance-6','x', ?)")) {
-                stmt.setObject(1, Collections.singletonMap("key1", "value1"));
-                stmt.execute();
-
-                try (ResultSet rs = s.executeQuery("select attributes from t_map")) {
-                    System.out.println(rs.next());
-                    System.out.println(rs.getObject(1));
+                try (ResultSet rs = stmt.executeQuery("SELECT * FROM " + TABLE_NAME)) {
+                    while (rs.next()) {
+                        System.out.println(rs.getDate(1) + ", " + rs.getInt(2) + ", " + rs.getString(3));
+                    }
                 }
             }
-        }
-    }
-
-    static void usedPooledConnection(String url) throws SQLException {
-        // connection pooling won't help much in terms of performance,
-        // because the underlying implementation has its own pool.
-        // for example: HttpURLConnection has a pool for sockets
-        HikariConfig poolConfig = new HikariConfig();
-        poolConfig.setConnectionTimeout(5000L);
-        poolConfig.setMaximumPoolSize(20);
-        poolConfig.setMaxLifetime(300_000L);
-        poolConfig.setDataSource(new ClickHouseDataSource(url));
-
-        HikariDataSource ds = new HikariDataSource(poolConfig);
-
-        try (Connection conn = ds.getConnection();
-                Statement s = conn.createStatement();
-                ResultSet rs = s.executeQuery("select 123")) {
-            System.out.println(rs.next());
-            System.out.println(rs.getInt(1));
-        }
```

Comment:
what other examples can we add? specifying clickhouse and client settings, adjusting log level?